apps/src/music/views/TimelineSimple2Events.tsx

```diff
@@ -116,7 +119,32 @@ interface TimelineSimple2EventsProps {
 const TimelineSimple2Events: React.FunctionComponent<
   TimelineSimple2EventsProps
 > = ({paddingOffset, barWidth, eventVerticalSpace, getEventHeight}) => {
-  const soundEvents = useMusicSelector(state => state.music.playbackEvents);
+  const soundEventsOriginal = useMusicSelector(
```

Comment:
Add comment that includes how these events are sorted as opposed to `useTimelineLayout2`.

Reply:
Good call.  Added more comments.